sdk/ml/azure-ai-ml/azure/ai/ml/entities/_inputs_outputs/input.py

```diff
@@ -216,7 +216,7 @@ def __init__(
         super(Input, self).__init__(type=type)
         # As an annotation, it is not allowed to initialize the name.
         # The name will be updated by the annotated variable name.
-        self.name = None
+        self.previous_name = None
```

Comment:
Let's call this: port_name or arg_name 

previous_name is hard to understand.

Reply:
replaced with arg_name